.pre-commit-config.yaml

```diff
@@ -37,12 +39,13 @@ repos:
     hooks:
       # update the NOT INSTALLABLE ADDONS section above
       - id: oca-update-pre-commit-excluded-addons
-      # - id: oca-fix-manifest-website
-      #   args: ["https://www.quartile.co"]
+      #- id: oca-fix-manifest-website
+      #  args: [""]
       - id: oca-gen-addon-readme
         args:
           - --addons-dir=.
           - --branch=16.0
+          - --org-name=qrtl
           - --repo-name=ymd-oca
           - --if-source-changed
```

Comment:
Remove this check. If not, this will override README.rst for all existing modules.

Reply:
Understand. I will delete line 48 and do copier update choose "Both" option.
and close this PR. Created a new one.